charts/spire/README.md.gotmpl

```diff
@@ -19,7 +19,7 @@
 
 | Dependency | Supported Versions |
 |:-----------|:-------------------|
-| SPIRE      | `1.5.3+`, `1.6.3+` |
+| SPIRE      | `1.8.4`            |
```

Comment:
I think this whole file may not be used anymore? Should we get rid of it?

Reply:
correct. This got replaced when we switched doc tool

Reply:
Filed https://github.com/spiffe/helm-charts-hardened/issues/87

Reply:
https://github.com/spiffe/helm-charts-hardened/pull/86